src/code-templates/services/order-service/entry-points/api/server.ts

```diff
@@ -7,16 +7,28 @@ import { defineRoutes } from "./routes";
 import { errorHandler } from "@practica/error-handling";
 import * as configurationProvider from "@practica/configuration-provider";
 import configurationSchema from "../../config";
+import { AsyncLocalStorage } from 'async_hooks';
+import { v4 } from 'uuid';
 
 let connection: Server;
+const asyncLocalStorage = new AsyncLocalStorage<Map<string, string>>();
+
+function requestId(req, res, next) {
+    asyncLocalStorage.run(new Map(), () => {
+      asyncLocalStorage.getStore()?.set("practicaRequestId", v4());
```

Comment:
Eventually this code will be used by various business apps, they probably don't want the name 'practica' in their code. Shall we call this just requestId